### PR TITLE
refactor: adapt types for newer `@endo/eventual-send`

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

Make Endo master integration pass CI again.

### CODE REVIEW

File: packages/wallet/api/src/lib-wallet.js [hunk 1]
```diff
@@ -726,7 +726,7 @@ export function makeWalletRoot({
         // eslint-disable-next-line no-use-before-define
         p = makeEmptyPurse(petnameForBrand, petnameForBrand, true);
       } else {
-        p = Promise.resolve();
+        p = Promise.resolve(undefined);
```
Comment (mhofman) [inline packages/wallet/api/src/lib-wallet.js:729]: I really dislike this about tsc promise's type handling.

File: scripts/get-packed-versions.sh [hunk 2]
```diff
@@ -27,12 +32,24 @@ yarn --silent workspaces info | jq -r '.[].location' | while read -r dir; do
   stem=$(echo "$name" | sed -e 's!^@!!; s!/!-!g;')
   file="$(pwd)/${stem}-v${version}.tgz"
 
+  # Clean up.
+  rm -f "${stem}"-v*.tgz
```
Comment (mhofman) [inline scripts/get-packed-versions.sh:36]: 👍

File: scripts/get-packed-versions.sh [hunk 3]
```diff
@@ -27,12 +32,24 @@ yarn --silent workspaces info | jq -r '.[].location' | while read -r dir; do
...
+
   # Create the tarball.
   yarn pack 1>&2
 
+  if $cache_bust; then
+    # Bust the cache!
```
Comment (mhofman) [inline scripts/get-packed-versions.sh:42]: Why not do this unconditionally?

**Review by mhofman — APPROVED** [review]
LGTM, but I'm wondering if we shouldn't just always append the hash of the package to the file?

File: scripts/get-packed-versions.sh (hunk 3 above)
Comment (michaelfig) [reply scripts/get-packed-versions.sh:42]: I'd be happy to make that change.  Will do before landing.

**Pull request merged**